apps/webstore/src/app/shared/components/products/quotable-design/quotable-design.component.ts

```diff
@@ -103,7 +114,29 @@ export class QuotableDesignComponent implements AfterViewInit, OnDestroy, OnInit
 
   ngOnChanges(changes: SimpleChanges): void {
     if (changes.design !== undefined) {
-      this.adjustInputDimensions();
+      setTimeout(() => {
+        this.adjustInputDimensions();
+      }, 100);
+    }
+  }
+
+  showAddTextButton(): boolean {
+    if (!this.design.showText && this.isMutable) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Just return the result of the condition...
```suggestion
    return !this.design.showText && this.isMutable
```